resources/public/api/conf/3.0/coding_out_status_retrieve.yaml

```diff
@@ -0,0 +1,75 @@
+get:
+  summary: Retrieve Coding Out Status
+  description: |
+    This endpoint enables you to retrieve opt-out of coding out status for a specified customer (identified by a National Insurance number) and tax year.
+    
+    ### Test data
+    Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.
+    
+    | Header Value (Gov-Test-Scenario)        | Scenario                                                           |
+    |-----------------------------------------|--------------------------------------------------------------------|
+    | N/A - DEFAULT                           | Simulates a successful response with latest values.                |
+    | BUSINESS_PARTNER_NOT_EXIST              | Simulates a scenario where the business partner does not exist.    |
+    | ITSA_CONTRACT_OBJECT_NOT_EXIST          | Simulates a scenario where the ITSA contract object does not exist.|
+
+  security:
+    - User-Restricted:
+        - read:self-assessment
+
+  tags:
+    - Opting out of Coding Out
+
+  parameters:
+    - $ref: "./common/headers.yaml#/components/parameters/acceptJson"
+    - $ref: "./common/headers.yaml#/components/parameters/authorizationReadScopeHeader"
+    - $ref: "./common/headers.yaml#/components/parameters/testScenarioHeader"
+    - $ref: "./common/pathParameters.yaml#/components/parameters/nino"
+    - $ref: "./common/pathParameters.yaml#/components/parameters/taxYear"
+
+  responses:
+    "200":
+      description: Success
+      headers:
+        X-CorrelationId:
+          $ref: "./common/headers.yaml#/components/parameters/correlationId"
+      content:
+        application/json:
+          schema:
+            $ref: "./schemas/retrieve_coding_out_status_response.json"
+          examples:
+            Request:
+              value:
+                $ref: "./examples/retrieveCodingOutStatus/retrieve_coding_out_status_response.json"
+
+    "400":
+      description: Bad request
+      content:
+        application/json:
+          schema:
+            $ref: './schemas/error-response.yaml'
+          examples:
+            FORMAT_NINO:
+              $ref: "./common/errors.yaml#/components/examples/formatNino"
+#            FORMAT_TAX_YEAR:
```

Comment:
If this isnt needed can we remove it from the OAS? The tech spec shows that this error is present

Reply:
It was under discussion whether or not to add this error for this endpoint, It's confirmed now that we are keeping it. I've uncommented it in latest commit.